src/vendor/wordpress/_wordpress.scss

```diff
@@ -0,0 +1,24 @@
+@use "../../mixins/button";
+@use "../../design-tokens/sizes.yml";
+
+/**
+ * Gutenberg block: Button
+ * Applies our pattern library styles to buttons generated via Gutenberg blocks.
+ *
+ * 1. The block editor allows editors to adjust button border radius. This style
+ *    is applied via an inline style on the `wp-block-button__link` element. In
+ *    order to prevent unwanted border-radius we apply an `!important` tag.
+ */
+
+.wp-block-button {
+  display: inline-block;
```

Comment:
I'm assuming this is so multiple buttons will not break to new lines.

Did you consider using `display: flex` on `wp-block-buttons` with padding for nicer gaps between multiple buttons?

Reply:
> I'm assuming this is so multiple buttons will not break to new lines.

Correct. My intent was to match the presentation and layout of our existing `c-button` pattern as much as possible. 

> Did you consider using `display: flex `on `wp-block-buttons` with padding for nicer gaps between multiple buttons?

I had considered using `display: flex` with a gap, but did not consider using padding. 🤔 

If you feel like adding `display: flex` with padding is the preferred approach here, I need to think though how the padding would work.

Reply:
Sorry, I should have said "whitespace" (did not mean to suggest the solution actually must use `padding`).